### PR TITLE
Collect user feedback

### PR DESCRIPTION
**Pull request by qrtt1**
**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**

feature

**What this PR does / why we need it**:

collect user feedback

**Which issue(s) this PR fixes**:

ch20220

**Notes**:

* tag: `ch20220-a3df22`
* this is a `CE Only` feature

### CODE REVIEW

File: packages/client/src/components/share/tracker.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    notification.open({
+      key: 'prompt-notification',
+      message: `${title}`,
+      placement: 'bottomRight',
+      duration: 0,
+      description: <CommunityList userTakeAction={userTakeAction} state={collectedState} setState={setCollectedState} />
+    });
+
+    return (
+      <div>
```
Comment (popcornylu) [inline packages/client/src/components/share/tracker.tsx:133]: return null or `<></>`

File: packages/client/src/components/share/tracker.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import { useLocation } from 'react-router-dom';
+import { useEffect, useState } from 'react'
+import { useLocalStorage } from 'hooks/useLocalStorage';
+import { Modal, Button, Icon, notification } from 'antd';
+import React from 'react';
```
Comment (aar0nTw) [inline packages/client/src/components/share/tracker.tsx:5]: Don't import same package twice, `import React, { useEffect, useState} from 'react'`

File: packages/client/src/components/share/tracker.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    </div>
+  )
+}
+
+const PageChangeTracker = (props) => {
+  if (primehubCE !== true) {
+    return (<></>)
+  }
+
+  const analytics = window['analytics'] || null;
```
Comment (aar0nTw) [inline packages/client/src/components/share/tracker.tsx:57]: use window.analytics

File: packages/client/src/components/share/tracker.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+    // close all prompt
+    setDisplay({ show: false, type: display.type });
+    notification.close('prompt-notification');
+    return true;
+  }
+
+  const title: string = 'Talk to PrimeHub';
+  if (display.show !== true) {
+    return (<></>)
```
Comment (aar0nTw) [inline packages/client/src/components/share/tracker.tsx:88]: pls install prettier for ur editor

**Review by aar0nTw — CHANGES_REQUESTED** [review]

File: packages/client/src/components/share/tracker.tsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
+import { useLocation } from 'react-router-dom';
+import { useEffect, useState } from 'react'
+import { useLocalStorage } from 'hooks/useLocalStorage';
+import { Modal, Button, Icon, notification } from 'antd';
+import React from 'react';
+
+const updatePageChange = (state, setState, setDisplay: Function) => {
+
```
Comment (aar0nTw) [inline packages/client/src/components/share/tracker.tsx:8]: `const {collected, location, pageChanges} = state;`

File: packages/client/src/components/share/tracker.tsx [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    return (<></>)
+  }
+
+  const analytics = window['analytics'] || null;
+  if (analytics == null) {
+    return (<></>)
+  }
+
+  const location = useLocation();
+  const [display, setDisplay] = useState({ show: false, type: 'modal' });
```
Comment (aar0nTw) [inline packages/client/src/components/share/tracker.tsx:63]: u should separate to different states.

**Review by aar0nTw — CHANGES_REQUESTED** [review]

**Comment by poga** [comment]
Wording:

Title: `Thanks for using PrimeHub! Any thing you want to share?`

link to twitter: `Share your experience to us on Twitter`
link to discord: `Join our Discord community`

**Comment by qrtt1** [comment]
@poga 

* prompt will not close automatically
* update new wording

image

image

**Review by aar0nTw — APPROVED** [review]
LGTM

**Comment by popcornylu** [comment]
LGTM.

**Pull request merged**